tests/hardware/plugins/stepper_motor/test_st10_controller.py

```diff
@@ -357,39 +357,33 @@ def test_is_moving(
 @pytest.mark.parametrize(
     "step,response,raises",
     chain(
-        [(step, f"SP={step}", does_not_raise()) for step in range(0, 250, 50)],
-        [(4, "SP=hello", pytest.raises(ST10ControllerError))],
+        [(step, f"IP={step}", does_not_raise()) for step in range(0, 250, 50)],
+        [(4, "IP=hello", pytest.raises(ST10ControllerError))],
     ),
 )
 @patch(
     "finesse.hardware.plugins.stepper_motor.st10_controller.ST10Controller.is_moving",
     new_callable=PropertyMock,
 )
-def test_get_step_not_moving(
+def test_get_step(
     is_moving_mock: PropertyMock,
     step: int,
     response: str,
     raises: Any,
     dev: ST10Controller,
 ) -> None:
-    """Test getting the step property when the motor is stationary."""
+    """Test getting the step property."""
+    # When the motor is stationary:
     is_moving_mock.return_value = False
     with read_mock(dev, response):
         with raises:
             assert dev.step == step
 
-
-@patch(
-    "finesse.hardware.plugins.stepper_motor.st10_controller.ST10Controller.is_moving",
-    new_callable=PropertyMock,
-)
-def test_get_step_moving(
-    is_moving_mock: PropertyMock,
-    dev: ST10Controller,
-) -> None:
-    """Test getting the step property when the motor is moving."""
+    # When the motor is moving:
     is_moving_mock.return_value = True
```

Comment:
We don't actually need to mock `ST10Controller.is_moving` anymore because it's not called by `ST10Controller.step`